diff --git a/out/util/yarn.js b/out/util/yarn.js
index fd319f62c8bbf5b8c48a1d70c1c7efa92f75e25f..59d2c4eae8477bb59871f8cd2a37ac3fb215263a 100644
--- a/out/util/yarn.js
+++ b/out/util/yarn.js
@@ -61,7 +61,7 @@ async function installOrRebuild(config, appDir, options, forceInstall = false) {
   };
   let isDependenciesInstalled = false;
 
-  for (const fileOrDir of ["node_modules", ".pnp.js"]) {
+  for (const fileOrDir of ["node_modules", "../../.pnp.js"]) {
     if (await (0, _fsExtra().pathExists)(path.join(appDir, fileOrDir))) {
       isDependenciesInstalled = true;
       break;
